drivers/hmis/app/graphql/types/forms/pick_list_option.rb

```diff
@@ -35,7 +35,7 @@ def self.options_for_type(pick_list_type, user:, project_id: nil)
         state_options.map do |obj|
           {
             code: obj['abbreviation'],
-            label: "#{obj['abbreviation']} - #{obj['name']}",
+            # label: "#{obj['abbreviation']} - #{obj['name']}",
```

Comment:
Switching state picker to a small width input with only abbreviations in the dropdown